compiler/shared/main/scala/mlscript/compiler/mono/Monomorph.scala

```diff
@@ -22,7 +22,7 @@ import mlscript.compiler.printer.ExprPrinter
 import mlscript.compiler.mono.specializer.BoundedExpr
 import mlscript.compiler.mono.specializer.{MonoValue, TypeValue, ObjectValue, UnknownValue, FunctionValue, VariableValue}
 import mlscript.{MonoVal, TypeVal, ObjVal, FuncVal, LiteralVal, PrimVal, VarVal, TupVal, UnknownVal, BoundedTerm}
-
+ 
```

Comment:
??

Reply:
It's good practice to review your diffs before pushing your commits.

Reply:
Noted. I've turned on show whitespace diffs in my IDE.